zpeclient.go

```diff
@@ -81,6 +87,22 @@ func NewZpeClient(crIndexInformer cache.SharedIndexInformer, log Logger) *Cache
 			privateCache.deleteObj(item)
 		},
 	})
+	cmIndexInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			err := privateCache.parseUpdateTime(obj)
+			if err != nil {
+				log.Println("Unable to convert informer store item into AthenzDomain type.")
+				return
+			}
+		},
+		UpdateFunc: func(oldObj, newObj interface{}) {
+			err := privateCache.parseUpdateTime(newObj)
```

Comment:
since this configmap is updated relatively less frequently, should we make all the compute that we do in `checkUpdateTime` to determine the time difference to `parseUpdateTime` and store result in a boolean (cache-status: active/stale) that `checkUpdateTime` could use ?